build/yaml/cleanupResources/cleanupResources.yml

```diff
@@ -136,6 +137,32 @@ stages:
                 Write-Host "No pre-existing $(INTERNALAPPSERVICEPLANPYTHONNAME) resource found."
               }
 
+- stage: "Delete_Virtual_Network"
```

Comment:
The stages are wrong, because they are all scrambled, and this happens.
![image](https://user-images.githubusercontent.com/62260472/124167839-b86dc900-da7a-11eb-8cf2-c4982a030694.png)

Reply:
Fixed 👍🏻 
![image](https://user-images.githubusercontent.com/54330145/124286408-4b633d80-db25-11eb-8a05-51263b063d7f.png)